app/models/spotlight/solr_document_sidecar.rb

```diff
@@ -6,9 +6,9 @@ class SolrDocumentSidecar < ActiveRecord::Base
 
     acts_as_taggable
 
-    belongs_to :exhibit, required: true
+    belongs_to :exhibit, optional: false
```

Comment:
Is this addressing a rubocop rule?

Reply:
Yeah, back around Rails 5 they flipped from `required` to `optional`.